content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -110,25 +100,13 @@ Follow our instructions on how to
     > say `<pending>`. You will need to access it using the service
     > NodePort, or use port-forwarding instead.
 
-1.  Ensure the corresponding Kubernetes pods are deployed and all containers
-    are up and running: `istio-pilot-*`, `istio-ingressgateway-*`,
-    `istio-egressgateway-*`, `istio-policy-*`, `istio-telemetry-*`,
-    `istio-citadel-*`, `prometheus-*`, `istio-galley-*`, and, optionally,
-    `istio-sidecar-injector-*`.
+1.  Ensure the corresponding Kubernetes pods are deployed and all containers: `istio-citadel-*`,
+    `istio-engressgateway-*`, `istio-galley-*`, `istio-ingress-*`, `istio-ingressgateway-*`,
+    `istio-pilot-*`, `istio-policy-*`, `istio-statsd-prom-bridge-*`, `istio-telemetry-*`,
+    `prometheus-*`, and optionally have `istio-sidecar-injector-*`:
```

Comment:
Grammar indeed. Pay close attention to the commas and the order of the words, your fix is not correct. The correct text in full: 
1. Ensure the corresponding Kubernetes pods are deployed and all containers, `istio-citadel-*`,
    `istio-engressgateway-*`, `istio-galley-*`, `istio-ingress-*`, `istio-ingressgateway-*`,
    `istio-pilot-*`, `istio-policy-*`, `istio-statsd-prom-bridge-*`, `istio-telemetry-*`,
    `prometheus-*`, and, optionally, `istio-sidecar-injector-*`, have a `STATUS`of `Running`: